include/mbedtls/config_psa.h

```diff
@@ -515,12 +524,18 @@ extern "C" {
 
 #if defined(PSA_WANT_ALG_CMAC)
 #if !defined(MBEDTLS_PSA_ACCEL_ALG_CMAC) || \
-    defined(PSA_HAVE_SOFT_BLOCK_CIPHER)
+    defined(PSA_HAVE_SOFT_BLOCK_CIPHER) || \
+    defined(PSA_HAVE_SOFT_PBKDF2_CMAC)
```

Comment:
I think you're right: the PBKDF2-AES-CMAC code should work if AES-CMAC is provided by a driver. I'm [not sure if CMAC drivers are working](https://github.com/Mbed-TLS/mbedtls/issues/8314) though...

Since you're working on https://github.com/Mbed-TLS/mbedtls/pull/8346, would you mind checking if this exclusion can be removed, i.e. does the accelerated-CMAC driver work without `|| defined(PSA_HAVE_SOFT_PBKDF2_CMAC)` here and for AES?